src/options.rs

```diff
@@ -60,24 +55,22 @@ impl ModelOptions {
         }
     }
 
-    pub fn with_invert_colors(mut self, color_inversion: ColorInversion) -> Self {
+    /// Sets the color inversion setting.
+    pub fn set_invert_colors(&mut self, color_inversion: ColorInversion) {
```

Comment:
I did change this to match `set_orientation`. Having two different methods of settings fields in one type seemed confusing.